src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -462,17 +463,16 @@ await supersededPublications
                         }
                         else
                         {
-                            var latestVersion = await _releaseVersionRepository
-                                .GetLatestReleaseVersionForParent(
-                                    seriesItem.ReleaseId!.Value,
-                                    publishedOnly: true);
+                            var latestVersion = await _context.ReleaseVersions
+                                .LatestReleaseVersion(seriesItem.ReleaseId!.Value, publishedOnly: true)
+                                .SingleOrDefaultAsync();
```

Comment:
I think a comment above this like `// prefer getting the latest published version over a more recent draft version if one exists` might help what's going off here when read in context together with the block below `// if no published version, look for an unpublished version`.

I was a bit confused at first, wondering why we don't just get the latest release version.